lib/shopify_api/resources/draft_order.rb

```diff
@@ -6,5 +6,9 @@ def send_invoice(draft_order_invoice = ShopifyAPI::DraftOrderInvoice.new)
       resource = post(:send_invoice, {}, draft_order_invoice.encode)
       ShopifyAPI::DraftOrderInvoice.new(ShopifyAPI::DraftOrder.format.decode(resource.body))
     end
+
+    def complete(options = {})
+      load_attributes_from_response(patch(:complete, {}, options.to_json))
```

Comment:
However, I did run into this issue (see failing tests below).

I'm thinking the only way to properly fix this is to change the endpoint itself to a put.